tests/integration_tests/db_engine_specs/trino_tests.py

```diff
@@ -166,3 +168,31 @@ def test_auth_custom_auth_denied(self):
             f"For security reason, custom authentication '{auth_method}' "
             f"must be listed in 'ALLOWED_EXTRA_AUTHENTICATIONS' config"
         )
+
+    def test_convert_dttm(self):
+        dttm = self.get_dttm()
+
+        self.assertEqual(
```

Comment:
Using `assert` rather than `self.assertEqual` is preferred in our `pytest`/`unittest` hybrid world.